_data/docs.yml

```diff
@@ -123,6 +123,16 @@ apis:
       legacy: 1
       stable: 1
       nightly: 1
+  pylibcugraphops:
+    name: cuGraph-Ops
+    path: pylibcugraphops
+    desc: 'cuGraph-Ops aims to be a low-level, framework agnostic library providing commonly used computational operators for GNNs and other graph operations.'
+    ghlink: https://github.com/rapidsai/cugraph-ops
```

Comment:
```suggestion
```

`cugraph-ops` is a private repository, so we shouldn't post links to it publicly